pkgs/development/compilers/ligo/default.nix

```diff
@@ -80,23 +90,23 @@ coq.ocamlPackages.buildDunePackage rec {
     irmin-pack
     ezjsonm
     data-encoding
+    pure-splitmix
+    zarith_stubs_js
   ];
 
   checkInputs = [
     cacert
-    coq.ocamlPackages.ca-certs
+    ocamlPackages.ca-certs
   ];
 
   doCheck = false; # Tests fail, but could not determine the reason
 
-  patches = [ ./ligo.patch ]; # fix for core >= 0.15.0
```

Comment:
Hoping that we land https://gitlab.com/ligolang/ligo/-/merge_requests/2061 before the release, I'm working with the devs to make it happen.

Reply:
This has landed